lib/containers/k8s.pm

```diff
@@ -47,17 +47,21 @@ sub uninstall_k3s {
 }
 
 sub install_kubectl {
-    if (is_sle) {
-        # kubectl is in the container module
-        add_suseconnect_product(get_addon_fullname('contm'));
-        # SLES-15SP2+ ships a specific kubernetes client version. Older versions hold a version-independent kubernetes-client package.
-        if (is_sle(">15-SP1")) {
-            zypper_call("in kubernetes1.18-client");
+    if (script_run("which kubectl") != 0) {
+        if (is_sle) {
+            # kubectl is in the container module
+            add_suseconnect_product(get_addon_fullname('contm'));
+            # SLES-15SP2+ ships a specific kubernetes client version. Older versions hold a version-independent kubernetes-client package.
+            if (is_sle(">15-SP1")) {
+                zypper_call("in kubernetes1.18-client");
+            } else {
+                zypper_call("in kubernetes-client");
+            }
         } else {
             zypper_call("in kubernetes-client");
         }
     } else {
-        zypper_call("in kubernetes-client");
+        record_info('kubectl already', 'The kubectl package is already installed.');
```

Comment:
```suggestion
        record_info('kubectl', 'The kubectl package is already installed.');
```
`kubectl already` alone sounds a bit weird :) 

Also, what about printing the version in another record_info ?

Reply:
Done.